diff --git a/MODULE.bazel b/MODULE.bazel
new file mode 100644
index 000000000..8dc4a268b
--- /dev/null
+++ b/MODULE.bazel
@@ -0,0 +1,16 @@
+module(
+    name = "googleapis",
+    version = "0.0.0-20250120-318818b.bzlmod.2",
+    bazel_compatibility = [">=7.4.0"],
+    repo_name = "com_google_googleapis",
+)
+
+bazel_dep(
+    name = "protobuf",
+    version = "29.3",
+    repo_name = "com_google_protobuf",
+)
+bazel_dep(name = "rules_proto", version = "7.1.0")
+
+imports = use_extension(":extensions.bzl", "imports")
+use_repo(imports, "com_google_googleapis_imports")
